Jenkinsfile

```diff
@@ -30,66 +33,100 @@ pipeline {
             steps {
                 edgeXSetupEnvironment()
                 edgeXDockerLogin(settingsFile: env.MVN_SETTINGS)
+                edgeXSemver 'init'
+                script {
+                    def semverVersion = edgeXSemver()
+                    env.setProperty('VERSION', semverVersion)
+                    sh 'echo $VERSION > VERSION'
+                    stash name: 'semver', includes: '.semver/**,VERSION', useDefaultExcludes: false
+                }
             }
         }
 
         stage('Multi-Arch Build') {
             // fan out
             parallel {
-                stage('Test amd64') {
+                stage('Build amd64') {
                     agent {
-                        dockerfile {
-                            filename 'Dockerfile.build'
-                            label 'centos7-docker-4c-2g'
-                            args '-u 0:0 -v /var/run/docker.sock:/var/run/docker.sock' // needed for go mod cache
-                        }
+                        label 'centos7-docker-4c-2g'
                     }
-                    steps {
-                        sh 'make test'
-                        // edgeXCodecov('app-functions-sdk-go-codecov-token')
-                        sh 'make docker'
+                    stages {
+                        stage('Docker Build') {
+                            steps {
+                                unstash 'semver'
+                                
+                                sh 'echo Currently Building version: `cat ./VERSION`'
+
+                                script {
+                                    image_amd64 = docker.build(
+                                        'docker-app-service-configurable',
+                                        "--build-arg MAKE='test build' --label 'git_sha=${env.GIT_COMMIT}' ."
+                                    )
+
+                                    // edgeXCodecov('app-functions-sdk-go-codecov-token')
```

Comment:
Remove if no longer needed. Is CodeCov enabled?

Reply:
I need to check with eball to see if they have configured the token

Reply:
I will remove for now...the way we have implemented the test and build steps do not allow codecov to gather the test files needed. Plus the make test target would need to be updated to: `GO111MODULE=on go test -coverprofile=coverage.out ./...`